modals/README.md

```diff
@@ -18,74 +18,121 @@ implementation("com.decathlon.vitamin.compose:modals:<versions>")
 ```kotlin
 object VitaminModals {
     @Composable
-    fun Primary2(
+    fun Primary(
         content: @Composable (() -> Unit),
         onDismissRequest: () -> Unit,
         modifier: Modifier = Modifier,
         title: String? = null,
         painter: Painter? = null,
-        buttonsOrientation: ModalButtonsOrientation = ModalButtonsOrientation.HORIZONTAL,
         contentScrollState: ScrollState = rememberScrollState(),
         colors: ModalColors = VitaminModalColors.primary(),
         sizes: ModalSizes = VitaminModalSizes.medium(),
-        positiveButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
-        negativeButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
-        neutralButton: (@Composable VitaminModalButtons.() -> Unit)? = null,
+        button: (@Composable VitaminModalButtons.() -> Unit)? = null,
     )
 }
 ```
 
-The primary modal used to provide critical information or ask for decision.
+This Primary modal implementation used to provide information.
 
 The minimal usage of the component is callback to be notified when the user need to dismiss the modal.
 
 ```kotlin
 VitaminModals.Primary(
     content = {
-        Text(
-            text = "Simple information displayed to the user"
-        )
+        Text(text = "Simple information displayed to the user")
     },
     onDismissRequest = {},
-    modifier = Modifier,
-    title = "Amazon",
-    painter = rememberVectorPainter(VitaminIcons.Line.MapPin),
-    contentScrollState = rememberScrollState(),
-    positiveButton = {
+    painter = rememberVectorPainter(image = VitaminIcons.Line.MapPin),
+    title = "title",
+    button = {
         VitaminModalButtons.Primary(
-            text = "Positive",
+            text = "Got it",
             onClick = {}
         )
+    }
+)
+```
+
+Parameters | Descriptions
+-- | --
+`content: @Composable (() -> Unit)` | The content to be displayed below the title
+`onDismissRequest: () -> Unit` | The callback to be called when the user need to dismiss the modal
+`modifier: Modifier = Modifier` | The `Modifier` to be applied to the component
+`title: String? = null` | The optional title to be displayed at the top of the modal
+`painter: Painter? = null` | The optional leading icon to be displayed at the start of the title
+`contentScrollState: ScrollState = rememberScrollState()` | The optional scroll state to observe the scrolling
+`colors: ModalColors = VitaminModalColors.primary()` | The optional sizes used to define colors of icon, title, content and background of the modal
+`sizes: ModalSizes = VitaminModalSizes.medium()` | The optional sizes used to define modal padding, spacers sizes and icon size
+`button: (@Composable VitaminModalButtons.() -> Unit)? = null` | The optional button to do the acknowledgement action
+
+### Primary
```

Comment:
Weird to have two sub titles named "Primary". Probably better to give more information than just "Primary"